### PR TITLE
UNI-161: Add File List component

### PR DESCRIPTION
**Pull request by lscheinkman**
@marionleborgne Check d7c5422 for the CSV parsing.

@brev Please review

### CODE REVIEW

**Comment by lscheinkman** [comment]
Merging now for @marionleborgne 
@brev I appreciate if you could still take a look

File: unicorn/frontend/lib/FileServer.js [hunk 1]
```diff
@@ -43,23 +43,177 @@ const FILE_PATH = path.join('frontend', 'samples'); // @TODO move path to config
 /**
  *
  */
-var FileServer = function () {
+var FileServer = function() {
+  this.FILE_PATH = SAMPLES_FILE_PATH;
```
Comment (brev) [inline unicorn/frontend/lib/FileServer.js]: can line 47 be removed?

**Comment by brev** [comment]
:+1: looking great @lscheinkman

File: unicorn/frontend/lib/FileServer.js (hunk 1 above)
Comment (lscheinkman) [inline unicorn/frontend/lib/FileServer.js]: Yes. I will do that

**Pull request merged**